modules/renter/streamer.go

```diff
@@ -0,0 +1,141 @@
+package renter
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"math"
+	"time"
+
+	"github.com/NebulousLabs/errors"
+)
+
+type (
+	// streamer is a io.ReadSeeker that can be used to stream downloads from
+	// the sia network.
+	streamer struct {
+		file   *file
+		offset int64
+		r      *Renter
+	}
+)
+
+// min is a helper function to find the minimum of multiple values.
+func min(values ...uint64) uint64 {
+	min := uint64(math.MaxUint64)
+	for _, v := range values {
+		if v < min {
+			min = v
+		}
+	}
+	return min
+}
+
+// Streamer create a io.ReadSeeker that can be used to stream downloads from
```

Comment:
should be `Streamer creates a`

Reply:
and maybe `an`, if you pronounce the `io` when reading `io.ReadSeeker` :P